### ISSUE TITLE
ReadModel getters not working in queries by ID

### ISSUE THREAD

**Issue by javiertoledo** [boostercloud/booster#1312]
# Bug Report

## Current Behavior

Calculated fields exposed by read model getters are reportedly not working when using queries by ID. When the same read model is queried through the `ListReadModel`, they work normally.

## Expected behavior

Fields exposed by getters should be reachable no matter what kind of query is used.

## Possible Solution

I haven't looked into this yet, but I suspect that there might be an uncaught promise that is not resolving fast enough to fill the data in the response.

### Environment

- Booster version: v1.4.0

**Comment by melinahernandeztheam** [comment]
Hi, I have a real example of my application :) 

Current Behaviour:
```
QUERY:
query {
  RecipientUserReadModel(id: "b813fc41-15e4-495e-81aa-18879beddb45") {
    id
    firstName
    relativesIds
    relatives{firstName}
 }
}
RESPONSE:
{
  "data": {
    "RecipientUserReadModel": {
      "id": "b813fc41-15e4-495e-81aa-18879beddb45",
      "firstName": "Other Name",
      "relativesIds": [
        "f8963eb0-25e1-4056-a065-cc806778330A",
        "369c2fac-941c-4775-9b58-c81fe83df5db"
      ],
      "relatives": null,
    }
  }
}
```

```
{
  ListRecipientUserReadModels(
    filter: {}
    limit: 1
    sortBy: {}
    afterCursor: "_____"
  ) {
    items {
      id
      firstName
      relativesIds
      relatives {firstName}
    }
    count
    cursor
  }
}
RESPONSE:
{
  "data": {
    "ListRecipientUserReadModels": {
      "items": [
        {
          "id": "b813fc41-15e4-495e-81aa-18879beddb45",
          "firstName": "Other Name",
          "relativesIds": [
            "f8963eb0-25e1-4056-a065-cc806778330A",
            "369c2fac-941c-4775-9b58-c81fe83df5db"
          ],
          "relatives": [
            {
              "firstName": "Relative name"
            },
            {
              "firstName": "Other Relative Name"
            }
          ]
        }
      ],
      "count": 1,
      "cursor": {
        "id": "1"
      }
    }
  }
}
```

**Comment by hendrik-lemon** [comment]
This is solved in `v1.11.1`
